projects/Inventaire/default.nix

```diff
@@ -23,11 +23,11 @@
       links = {
         settings-types = {
           text = "TypeScript file listing the intended types of every setting";
-          url = "https://codeberg.org/inventaire/inventaire/src/branch/main/server/types/config.ts";
+          url = "https://codeberg.org/inventaire/inventaire/src/commit/fd2083a2e19c690c198351d81ed34a8857e0ead6/server/types/config.ts";
         };
         settings-defaults = {
           text = "Default settings";
-          url = "https://codeberg.org/inventaire/inventaire/src/branch/main/config/default.cjs";
+          url = "https://codeberg.org/inventaire/inventaire/src/commit/fd2083a2e19c690c198351d81ed34a8857e0ead6/config/default.ts";
```

Comment:
I think these should be reverted since the current inventaire version has the `.cjs` extension

Reply:
I'd rather we don't do that, because we'd have to remember to keep this in sync. Let's keep the URLs how they were, for now.